Source/SwiftLintFramework/Rules/ColonRule.swift

```diff
@@ -15,6 +15,16 @@ public struct ColonRule: CorrectableRule, ConfigurationProviderRule {
 
     public init() {}
 
+    public var flexibleRightSpacing = false
+
+    public init(configuration: AnyObject) throws {
+        self.flexibleRightSpacing = configuration["flexible_right_spacing"] as? Int == 1
```

Comment:
No, this won't work. 

configuration contains `AnyObject` keys. 

Initial option was 
`configuration["flexible_right_spacing"] as? Bool == true`

But it won't convert the YAML `true` to a Swift `true`.